### PR TITLE
new comment for test,

### PR DESCRIPTION
**Pull request by hessam94**
installation ignored

### CODE REVIEW

File: .gitignore
Comment (hadign20) [inline .gitignore:1]: revert this change

File: CMG-C++/main.cpp
Comment (hadign20) [inline CMG-C++/main.cpp:1]: remove comment

File: .gitignore
Comment (hessam94) [reply .gitignore:1]: I reverted the file

File: CMG-C++/main.cpp
Comment (hessam94) [reply CMG-C++/main.cpp:1]: i changed the comment

File: .gitignore
Comment (hadign20) [reply .gitignore:1]: good

File: CMG-C++/main.cpp
Comment (hadign20) [reply CMG-C++/main.cpp:1]: no, remove the comment

Comment (hessam94) [reply CMG-C++/main.cpp:1]: i removed the comment, thanks
